### DISCUSSION TITLE
Integrate spark config with spline

### DISCUSSION THREAD

**Discussion by jackysrivastava** [AbsaOSS/spline#1336] [Q&A]
I'm looking for the integration steps that I can incorporate in my spark-config so that my pyspark code can run with getting the lineage data without having to define the lineage in the spark session. Also how should I run my pyspark code to get the lineage information from spline.

**Comment by wajda** [comment]
Did you check this?
https://github.com/AbsaOSS/spline-spark-agent?tab=readme-ov-file#initialization-codeless
